nltk/probability.py

```diff
@@ -244,18 +244,24 @@ def max(self):
             )
         return self.most_common(1)[0][0]
 
-    def plot(self, *args, **kwargs):
+    def plot(
+        self, *args, title="", cumulative=False, percents=False, show=True, **kwargs
+    ):
         """
         Plot samples from the frequency distribution
         displaying the most frequent sample first.  If an integer
         parameter is supplied, stop after this many samples have been
         plotted.  For a cumulative plot, specify cumulative=True.
         (Requires Matplotlib to be installed.)
 
-        :param title: The title for the graph
+        :param title: The title for the graph.
         :type title: str
-        :param cumulative: A flag to specify whether the plot is cumulative (default = False)
-        :type title: bool
+        :param cumulative: Whether the plot is cumulative. (default = False)
+        :type cumulative: bool
+        :param percents: Whether the plot uses percents instead of counts. Only when cumulative is True.
```

Comment:
@tomaarsen percents is just the unit, and can be used independently of whether the plot is cumulative, right?

Reply:
By which I mean something like the following:

```
        if cumulative:
            freqs = list(self._cumulative_frequencies(samples))
            ylabel = "Cumulative "
        else:
            freqs = [self[sample] for sample in samples]
            ylabel = ""


        if percents:
            freqs = [100 * f / self._N for f in freqs]
            ylabel += "Percents"
        else:
            ylabel += "Counts"
```

Reply:
I'll build it in when I have some time!

Reply:
I've added it. Thank you for your implementation, as well!